c10/util/Backtrace.cpp

```diff
@@ -25,6 +25,8 @@
 #endif
 #endif
 
+#include <link.h>
```

Comment:
This isn't portable so you'll have to figure out a set of macros to make this portable

Reply:
yes, just waiting for builds to yell at me...